.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,781 @@
+<!-- We should probably link to this file in #kirby-guild -->
+First of all - thank you for your interest in contributing to Kirby! 🥳
+<!-- Early on show that we are here to help if necessary and to educate people in how to get help -->
+If you at any point have any questions please read: [I have a question related to Kirby](#i-have-a-question-related-to-kirby)
+
+**Beware:** We reserve the right to close issues and pull requests that does not follow the steps outlined in [the contribution model](#the-contribution-model). This is not because we want to be mean or micro-manage the way things are done - but a project of this size requires some consistency in the way we cooperate in order to be manageable. 
+
+* [Introduction to the contribution model](#introduction-to-the-contribution-model)
```

Comment:
Consider document outline and heading levels - also we do not have any H1/top level heading 😄

Reply:
Done